yarn.lock

```diff
@@ -4,20 +4,20 @@
 
 "@adobe/css-tools@^4.3.1":
   version "4.4.2"
-  resolved "https://registry.yarnpkg.com/@adobe/css-tools/-/css-tools-4.4.2.tgz#c836b1bd81e6d62cd6cdf3ee4948bcdce8ea79c8"
```

Comment:
The registry has changed for all dependencies. Could you please use Yarn with the registry registry.yarnpkg.com instead of npmjs.org to avoid unnecessary changes?